core/comments/comment_view.ts

```diff
@@ -334,8 +337,12 @@ export class CommentView implements IRenderedElement {
    * elements to reflect the new size, and triggers size change listeners.
    */
   setSize(size: Size) {
-    const oldSize = this.size;
+    let oldSize = this.size;
     this.setSizeWithoutFiringEvents(size);
+    if (Size.equals(oldSize, this.size) && this.preResizeSize) {
```

Comment:
Why do we need to check that they're equal in addition to checking that the pre-resize size exists?

Reply:
If they're different, presumably setSize isn't being called as a result of a drag, and we should defer to the normally-calculated oldSize.

Reply:
Per offline discussion, removed this check since it is technically redundant.